src/components/ArticlePage/CommentsSection.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+
+const CommentsSection = ({ articleId }) => {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetch(
+      `https://back-end-nna.onrender.com/api/articles/${articleId}/comments`
+    )
+      .then((res) => res.json())
+      .then((data) => setComments(data.comments));
+  }, [articleId]);
```

Comment:
I would recommend switching from fetch to axios for all of your requests. This can also be extracted out and put in an api.js file. 

Have a look at these notes for a way to setup your requests as well:
https://l2c.northcoders.com/courses/sd-notes/front-end#sectionId=axios,step=intro